src/Appwrite/Utopia/Request/Filters/V18.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Appwrite\Utopia\Request\Filters;
+
+use Appwrite\Utopia\Request\Filter;
+
+class V18 extends Filter
+{
+    // Convert 1.5 params to 1.6
+    public function parse(array $content, string $model): array
+    {
+        switch ($model) {
+            case 'account.deleteMfaAuthenticator':
+                unset($content['otp']);
```

Comment:
I think there is more.. For example, functions accept scopes now.
Please do full comparison between 1.6.x and 1.5.8 release, and look at changes in all response models, and controllers (->param part)

Reply:
Ignore that, they are optional